recipes/arrow/all/conanfile.py

```diff
@@ -26,6 +26,7 @@ class ArrowConan(ConanFile):
         "encryption": [True, False],
         "filesystem_layer":  [True, False],
         "hdfs_bridgs": [True, False],
+        "runtime_simd_level": ["NONE", "SSE4_2", "AVX2", "AVX512", "MAX"],
```

Comment:
why "NONE" instead of `None`?

Reply:
Because the Cmake option expects "NONE" (as String), not None as empty value. Here is the official cmake: https://github.com/apache/arrow/blob/9aaf61c044189d4d15bd145380fb4e2a2a65138f/cpp/cmake_modules/DefineOptions.cmake#L121

Reply:
I would prefer lowercase values to be consistent in CCI recipes options, and add map values to what is expected in CMake.